Fix broken link in Cloudflare docs (#14017)
getsentry/sentry-docs: docs/platforms/javascript/common/configuration/integrations/fetchIntegration.mdx

---
title: FetchIntegration

> description: "Creates spans and attaches tracing headers to fetch requests in Cloudflare Workers. (default)"

Reviewer: ```suggestion
description: "A default integration that creates spans and attaches tracing headers to fetch requests in Cloudflare Workers."
```

supported:
  - javascript.cloudflare
---

<Alert>

> This integration only works in Cloudflare Workers runtime.

Reviewer: ```suggestion
This integration only works inside the Cloudflare Workers runtime.
```

</Alert>

_Import name: `Sentry.fetchIntegration`_

> This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).

Reviewer: ```suggestion
This integration is enabled by default. If you want to disable it, you can [modify your default integrations](./../#modifying-default-integrations).
```

> The `fetchIntegration` creates spans and attaches tracing headers to fetch requests in Cloudflare Workers.

Reviewer: ```suggestion
The `fetchIntegration` creates [spans](/concepts/key-terms/tracing/distributed-tracing/#traces-transactions-and-spans) and attaches tracing headers to fetch requests in Cloudflare Workers.
```
